nautobot/docs/installation/pip-install.md

```diff
@@ -0,0 +1,234 @@
+
+```no-highlight
+$ sudo python3 -m venv $NAUTOBOT_ROOT
+```
+
+Next, change ownership of `NAUTOBOT_ROOT` to the `nautobot` user:
+
+```no-highlight
+$ sudo chown -R nautobot:nautobot $NAUTOBOT_ROOT
+```
+
+## Sudo to nautobot
+
+Now that we've created the virtualenv, the remaining steps will be performed as the `nautobot` user. 
+
+!!! warning
+    Don't skip this step!!
+
+    It's important to install Nautobot as the `nautobot` user so that we don't have to worry about fixing permissions later.
+
+```no-highlight
+$ sudo su - nautobot
+```
+
+After becoming `nautobot`, we need to set the `NAUTOBOT_ROOT` environment variable again for this user (since it is a fresh session as far as the system is concerned):
+
+```
+$ export NAUTOBOT_ROOT=/opt/nautobot
+```
+
+### Add `NAUTOBOT_CONFIG` to `.bashrc`
+
+For bonus points, add this to `~/.bashrc` for `nautobot`:
+
+```no-highlight
+$ echo "export NAUTOBOT_ROOT=/opt/autobot" >> ~/.bashrc
+```
+
+From here on out, anytime you become `nautobot`, your `NAUTOBOT_ROOT` will be set automatically.
+
+## Activate the Virtual Environment
+
+To work inside a Python virtualenv, it must be activated. This makes sure that the version of Python you're using, as well any dependencies that you install remain isolated in this environment. 
+
+!!! info
+    If it helps, try to think of activating the virtualenv like entering a shell unique to Nautobot's application environment.
+
+```no-highlight
+$ source /opt/nautobot/bin/activate
+```
+
+Observe that after activating, your prompt will now be preceded with the name of the virtualenv (`nautobot`).
+
+## Prepare the Virtual Environment
+
+Before we install anything into the virtualenv, we want to make sure that Pip is running the latest version. This is neecessary because sometimes when a new virtualenv is created, a cached version of Pip is installed into it. (Yes, even after we've deliberately upgraded Pip at the system level!)
+
+We also want to deliberately install the `wheel` library which will tell Pip to always try to install wheel packages if they are available. A [wheel is a pre-compiled Python package](https://realpython.com/python-wheels/), which is quicker and safer to install because it does not require development libraries or `gcc` to be installed on your system just so that some more advanced Python libraries can be compiled. 
+
+```no-highlight
+$ pip3 install --upgrade pip wheel
+```
+
+## Install Nautobot
+
+Use Pip to install Nautobot:
+
+```no-highlight
+$ pip3 install nautobot
+```
+
+Great! We have a virtualenv ready for use by the `nautobot` user, so let's proceed to verifying the installation.
+
+## Verify your Nautobot Installation
+
+You should now have a fancy `nautobot-server` command in your environment. This will be your gateway to all things Nautobot! Run it to confirm the installed version of `nautobot`:
```

Comment:
```suggestion
You should now have a fancy `nautobot-server` command in your virtualenv. This will be your gateway to all things Nautobot! Run it to confirm the installed version of `nautobot`:
```